--- config.def.h	2018-03-08 20:11:42.119909072 +0000
+++ config.h	2018-03-10 10:54:28.618470010 +0000
@@ -85,24 +85,24 @@
 /* Terminal colors (16 first used in escape sequence) */
 const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#111111",
+	"#aa4450",
+	"#719611",
+	"#ff9800",
+	"#6688aa",
+	"#8f6f8f",
+	"#528b8b",
+	"#666666",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#181818",
+	"#ff6a6a",
+	"#b1d631",
+	"#87875f",
+	"#90b0d1",
+	"#8181a6",
+	"#87ceeb",
+	"#c1cdc1",
 
 	[255] = 0,
 
